Queue/Backend/Sentinel.php

```diff
@@ -33,7 +33,7 @@ protected function connect()
 
         foreach ($hosts as $index => $host) { // Sort or randomize as appropriate
             try {
-                $configuredClient = new \Credis_Client($host, $ports[$index], $timeout = 0.5, $persistent = false);
+                $configuredClient = new \Credis_Client($host, $ports[$index], $timeout = 0.5, $persistent = false, $database = 0, $this->password);
```

Comment:
Your change would always use `0` for the database instead of the configured database. It should be the following:
```suggestion
                $configuredClient = new \Credis_Client($host, $ports[$index], $timeout = 0.5, $persistent = false, $this->database, $this->password);
```